spec/brcobranca/retorno/cnab400/bradesco_spec.rb

```diff
@@ -34,13 +34,13 @@
     expect(pagamento.outros_recebimento).to eql('0000000000000')
     expect(pagamento.data_credito).to eql('150515')
     expect(pagamento.sequencial).to eql('000002')
+    expect(pagamento.cod_de_ocorrencia).to eql('02')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.